CMSIS/RTOS2/RTX/Source/rtx_lib.c

```diff
@@ -642,6 +642,15 @@ __WEAK void software_init_hook (void) {
   (void)osKernelInitialize();
 }
 
+#elif defined(__ICCARM__)
+
+extern void $Super$$__iar_data_init3(void);
+void $Sub$$__iar_data_init3(void)
```

Comment:
missing space before '(void)' (code style consistency withing this module)

Reply:
Fixed